app/views/shared/check_answers/_client_details.html.erb

```diff
@@ -77,7 +77,7 @@
       <%= row.with_action(
             text: t("generic.change"),
             href: providers_legal_aid_application_has_national_insurance_number_path(@legal_aid_application),
-            visually_hidden_text: "#{t('.aria_prefix')} {#{t('.nino')}",
+            visually_hidden_text: "#{t('.aria_prefix')} #{t('.nino')}",
```

Comment:
Do we need these string literal `#{}` for these translations?

Reply:
This is code that already existed. All I did was remove the unnecessary `{` as I noticed that it was not needed when I was going through the code base for this ticket.

Reply:
I have tried it without the string literals `#{}` and we do need it for `visually_hidden_text`. 
However, it's not needed for `visually_hidden_suffix`.